apps/allocations/app/index.js

```diff
@@ -1,57 +1,13 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import Aragon, { providers } from '@aragon/api'
+import { AragonApi } from '@aragon/api-react'
+import appStateReducer from './app-state-reducer'
 import App from './components/App/App'
+import '@babel/polyfill'
```

Comment:
this import for polyfill should be always the first line on the code, to make sure the transpilation is applied to the whole file

Reply:
Ok!